activerecord/lib/active_record/errors.rb

```diff
@@ -126,7 +126,7 @@ def initialize(adapter = nil, message: nil, table: nil, foreign_key: nil, target
           To resolve this issue, change the type of the `#{foreign_key}` column on `#{table}` to be :integer. (For example `t.integer #{foreign_key}`).
         EOM
       else
-        msg = <<-EOM
+        msg = +<<-EOM
```

Comment:
We can't use `+` because it don't work in Ruby 2.2

Reply:
Whoops, I thought `+` was available. `dup` will work instead.